fineract-provider/src/main/java/org/apache/fineract/notification/config/MessagingConfiguration.java

```diff
@@ -20,9 +20,11 @@
 
```

Comment:
@hero78 , As per our talk, I wouldn't expect change anything in this class.

Reply:
fixed